src/services/myMarkersStorage.js

```diff
@@ -0,0 +1,18 @@
+export function getMarkersFromLocalStorage() {
+  const myMarkers = JSON.parse(localStorage.getItem("markerData")) || [];
+  return myMarkers;
+}
+
+export function addMarkerToLocalStorage(markerData) {
+  const myMarkers = getMarkersFromLocalStorage();
+  myMarkers.push(markerData);
+  localStorage.setItem("markerData", JSON.stringify(myMarkers));
+}
+
+export function removeMarkerFromLocalStorage(tripName) {
+  const myMarkers = getMarkersFromLocalStorage();
+  const newMarkers = myMarkers.filter((marker) => {
+    return marker.name !== tripName;
+  });
+  localStorage.setItem("markerData", JSON.stringify(newMarkers));
+}
```

Comment:
The file `myMarkersStorage.js` looks exactly the same as `myTripsStorage.js ` this screams for a refactoring. Basically you need to make the key, to which the data is saved to in localStorage, variable.